assets/0017-SequenceDiagram-NegotiationPhase.puml

```diff
@@ -0,0 +1,55 @@
+' DIAGRAM #########################################
+' Negotiation phase messaging
+' #################################################
+@startuml
+' #################################################
+' SETTINGS: color settings of diagram
+' #################################################
+skinparam sequence {
+	BorderColor black
+	ArrowColor black
+	ActorBorderColor black
+	LifeLineBorderColor black
+	LifeLineBackgroundColor white
+	
+	ParticipantBorderColor black
+	ParticipantBackgroundColor white
+	ParticipantFontColor black
+	
+	ActorBackgroundColor white
+	ActorFontColor black
+}
+' #################################################
+' ACTORS
+' #################################################
+actor Maker as M
+' participant "app\nMaker" as TM
+' participant "Social\nMedia" as I
+' participant "app\nTaker" as TT
+actor Taker as T
+
+' allow parallelism
+!pragma teoz true
+
+' #################################################
+' DIAGRAM
+' #################################################
+group negotiation phase
+    T->M: offer request (from invite)
+    M->T: current offer
+    T->T: decision to take offer
+    T->M: take offer
+    alt
+        M->T: Acknowledge
+    else some kind of failure
+        M->T: \nmissing param,\noffer outdated,\nexpiries not accepted
+        note right #white
+            taker could send
+            another offer request
+        end note
+    end
+    T->T: default to role of Alice
```

Comment:
Why mentioning the default role but not default expiry? I'd say remove that from the diagram and just include the information that is exchanged during the early messages to make it clear.